master/master.go

```diff
@@ -33,6 +37,13 @@ func TerminateSlaves(svc *ec2.EC2, instances []*ec2.Instance) (*ec2.TerminateIns
 	return svc.TerminateInstances(params)
 }
 
+func SendTask(t *task.Task, i *ec2.Instance) (resp *http.Response, err error) {
```

Comment:
This looks a bit strange to me, the return types don't need to be named right? Or am I missing something...